### DISCUSSION TITLE
SDK 1.9 BT-747-Lot (3 out of 4 filled)

### DISCUSSION THREAD

**Discussion by luispozoWS** [OP-TED/eForms-SDK#1052] [Q&A] [answered]
Hello,

Why is there no rule in french when we create 4 groups that have the field BT-747-Lot but only 3 are filled ?
<rulecontext="/*/cac:ProcurementProjectLot[cbc:ID/@schemeName='Lot']/cac:TenderingTerms/ext:UBLExtensions/ext:UBLExtension/ext:ExtensionContent/efext:EformsExtension[$noticeSubType = '14']">
<assert id="BR-BT-00747-0020" role="ERROR" diagnostics="ND-NonUBLTenderingTerms_BT-747-Lot" test="count(efac:SelectionCriteria/cbc:CriterionTypeCode[@listName='selection-criterion']) > 0">rule|text|BR-BT-00747-0020</assert>

Best regards

**Comment by YvesJo** [comment, marked as answer]
Hi,
The rules are independent from the language used for edition. Groups left empty do not lead to XML element generation as eforms does not accept empty elements.
